plugins/org.eclipse.sirius.doc/doc/i18n_changes.html

```diff
@@ -74,6 +74,9 @@ <h3 id="MessageKeysAdded">Message Keys Added</h3>
 			<li>
 				<code>org.eclipse.sirius.diagram.ui.provider.Messages.MovePinnedElementsAction_toolTipText</code>
 			</li>
+			<li>
+				<code>org.eclipse.sirius.diagram.ui.provider.Messages.PinElementsEclipseAction_tooltip</code>
+			</li>
```

Comment:
is it an omission from another ticket or is it a mistake?

Reply:
I don't know, maybe a rebase issue. Removed.